tests/testthat/test-downloadModule.R

```diff
@@ -45,6 +45,7 @@ test_that("downloadModule downloads and unzips a parent module", {
   d_expected <- moduleMetadata("LCC2005", tmpdir)$childModules %>%
     c(m, "data", "testthat") %>% sort()
 
+#   expect_equal(length(f), 42)
   expect_equal(length(f), 44)
```

Comment:
This is the problem I found yesterday. I do not know what changes you made. Seems like 44 is for now.

Yong Luo

> On Feb 2, 2016, at 11:05 PM, Alex Chubaty <notifications@github.com> wrote:
> 
> In tests/testthat/test-downloadModule.R:
> 
> > @@ -45,6 +45,7 @@ test_that("downloadModule downloads and unzips a parent module", {
> >    d_expected <- moduleMetadata("LCC2005", tmpdir)$childModules %>%
> >      c(m, "data", "testthat") %>% sort()
> >  
> > +#   expect_equal(length(f), 42)
> >    expect_equal(length(f), 44)
> was this change from 42 to 44 a result of you adding 2 additional files? which files are added here?
> 
> —
> Reply to this email directly or view it on GitHub.
>